### DISCUSSION TITLE
Welcome to Nakamori (Kodi Plugin) Discussions!

### DISCUSSION THREAD

**Discussion by bigretromike** [bigretromike/nakamori#474] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Samadaeus** [comment]
Hey man, I like what you’re doing here solving a solid issue in our niche community. I have a question though, does the file have to be pre installed or is there a path link I can type in to kodi’s file manager and work off that way? I’m have Kodi on a firetv stick. I also should mention I’m running v.19.3

**Reply by da3dsoul** [reply]
Nakamori is a client plugin for Shoko Server. https://shokoanime.com/

**Reply by bigretromike** [reply]
@Samadaeus you have to download a repo.zip file to install it via file mamanger inside kodi, that way you can receive every later update.
If I remember correctly you should be good with installing this file: https://shokunin.monogatari.pl/repo-dev/repository.nakamori-dev/repository.nakamori-dev-1.0.0.zip

Later you will be able to install nakamori from nakamori repo inside kodi and you ready to go to link you nakamori client with shokoserver as @da3dsoul mention.
